### DISCUSSION TITLE
Cannot reverify my academic affiliation.

### DISCUSSION THREAD

**Discussion by SiamOfficial** [community/community#51293] [GitHub Education]
So I have been reverifying my academic affiliation to gain access to GitHub studentship. I have been trying to prove that I am a student with my school pay card (because my school has no school id card yet). A long ago, it did work but now it says that I haven't completed my profile and don't have the same name as on my pay card but I have set the same name already. My pay card language is not English as I live in Bangladesh and use Bengali. So please help me.

**Comment by samus-aran** [comment]
Hi @SiamOfficial 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Reply by SiamOfficial** [reply]
There is no English translation of my school pay card (as my school doesn't has a school id), I can try translating the picture of my pay card but GitHub wants to take a live picture. Last time, GitHub accepted it and gave me a studentship. But now it's throwing excuses. Even I changed my name to Bengali which is the language of the school pay card. Can't GitHub use its system to translate the image? Or it is too hard for them to do. Because of that issues, I cannot update my website. I either have to public my repository or need to upgrade. But as a student, I don't have the ability to buy a subscription. 😭
